src/physics/em/detail/LivermorePE.cu

```diff
@@ -54,8 +54,8 @@ __global__ void livermore_pe_interact_kernel(const LivermorePEPointers   pe,
 
     // Sample an element
     ElementSelector    select_el(material.material_view(),
-                                 LivermorePEMicroXsCalculator{pe, particle},
-                                 material.element_scratch());
+                              LivermorePEMicroXsCalculator{pe, particle},
+                              material.element_scratch());
```

Comment:
There's one more spot above (`PhysicsTrackView physics`...) that got accidentally unformatted in a previous PR.

Reply:
Turns out that one is a glitch in clang-format 😦 perhaps something to do with how the lines above are 80 characters, and the "preference" is set to 79. Options are:
- Remain as is, inconsistent to our eyes but consistent as far as clang-format "thinks." Ugly but reproducible.
- Add clang-format blocks around it. Ugly.
- Add a blank line above the physics track view. Not ugly, but inconsistent.

I've already spent way too much time on this 😂  so I vote for option 1, and hope that a future clang-format will fix.

Reply:
I prefer the first option too, keep clang-format happy and not worry about it any more 🙂

Reply:
Even if 0.1% of the code is uglier for it, I like clang-format because it means I can disavow all responsibility for how the code looks once the `.clang-format` is set up.

Reply:
Definitely agree here - also we've observed `clang-format` to have some unusual behavior with `__global__` etc on the CUDA side.

Bottom line - I think its fair and appropriate to just blame `clang-format`.

Reply:
I prefer the term "art critic" 😉